src/main/java/hudson/plugins/logparser/LogParserAction.java

```diff
@@ -59,7 +64,7 @@ public static String getUrlNameStat() {
         return urlName;
     }
 
-    public AbstractBuild<?, ?> getOwner() {
+    public Run<?, ?> getOwner() {

```

Comment:
@oleg-nenashev I'd like to talk wih the maintainer about it. The last release of this plugin was on **15 Dec 2010**. Do you think make sense?